README.md

```diff
@@ -4,3 +4,21 @@ Vulkan users. It is meant to facilitate triaging, discussion and routing of
 issues that don't necessarily fit under the scope of a single project within
 the Vulkan ecosystem.
 
+# Map of Vulkan Ecosystem projects
+Vulkan ecosystem is a big place, but there's a lot out there thats useful! In no particular order...
+
+## RenderDoc
+RenderDoc - a graphics debugger, currently available for Vulkan, D3D11, D3D12, and OpenGL development on Windows 7 - 10 and Linux.
+Contacts: [baldurk@baldurk.org](mailto:baldurk@baldurk.org), [#renderdoc on freenode IRC](https://kiwiirc.com/client/irc.freenode.net/#renderdoc)
+Main site: https://renderdoc.org/
+Code: https://github.com/baldurk/renderdoc
```

Comment:
Code -> Source code
It is also an issue tracker. Is that worth noting here?

Reply:
I think we can just call it "repository" and leave that implicit instead. But might be worth pointing out for projects not on GitHub or when the issue tracker is separate from the main repository.